pandas/io/gbq.py

```diff
@@ -266,7 +270,10 @@ def sizeof_fmt(num, suffix='b'):
 
     def get_service(self):
         import httplib2
-        from apiclient.discovery import build
+        try:
+            from googleapiclient.discovery import build
```

Comment:
IOW the ci should be testing the original one & the new one (in different builds); so at least import checking should work.